apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -51,6 +39,12 @@ const WebinarQuestionsTemplate = ({
     const sortedQuestions = [...data].sort(
       (a, b) => b.id.createdAt.getTime() - a.id.createdAt.getTime()
     );
```

Comment:
Could we add here a `filter` functions that filters the questions hidden by other users so we don't have them inside the list?